src/main/java/com/example/generativeai/GenerativeaiApplication.java

```diff
@@ -2,12 +2,15 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
 
 @SpringBootApplication
+@EntityScan(basePackages = {"com.example.generativeai.entity"})
```

Comment:
No need to add this annotation. Entities will be scanned by default with help of `@SpringBootApplication`